packages/design-tokens/README.md

```diff
@@ -22,9 +22,30 @@ yarn add @sumup/design-tokens
 
 ## Usage
 
-The package currently exports a single `light` theme that is meant to be used with SumUp's React component library, [Circuit UI](https://github.com/sumup-oss/circuit-ui/tree/main/packages/circuit-ui). Pass the theme to Emotion's [`ThemeProvider`](https://emotion.sh/docs/theming):
+The [legacy JavaScript theme](#legacy-javascript-theme) for Emotion.js is being replaced by [CSS custom properties](#css-custom-properties) (aka CSS variables) to improve performance and compatibility with other frameworks. We recommend installing the [`@sumup/eslint-plugin-circuit-ui`](https://circuit.sumup.com/?path=/docs/packages-eslint-plugin-circuit-ui--docs) ESLint plugin and turning on the `prefer-custom-properties` and `no-invalid-custom-properties` rules to help with the migration.
 
-```jsx
+Refer to the [theme documentation](https://circuit.sumup.com/?path=/docs/features-theme--docs) for a full reference of the available tokens.
+
+### CSS custom properties
+
+The `@sumup/design-tokens/light.css` CSS file contains all CSS custom properties that are used by Circuit UI. Import it globally in your application, such as in Next.js' `_app.tsx` file:
+
+```tsx
+// _app.tsx
+import '@sumup/design-tokens/light.css';
```

Comment:
For switching between light and dark, how will that work?

Reply:
I'm not sure yet. There's still a lot of uncertainty around the dark theme (Will it follow the system settings? Will users be able to set their preferred one? Will this be built into `@sumup/design-tokens` or inside each app?), so I consciously left this out of scope. We may need to adjust the API in the future, but that would be a small breaking change.